tools/compile-compose.c

```diff
@@ -48,12 +48,8 @@ usage(FILE *fp, char *progname)
             " --file FILE\n"
             "    Specify a Compose file to load\n"
             " --locale LOCALE\n"
-            "    Specify the locale directly\n"
-            " --locale-from-env\n"
```

Comment:
The reason for adding this was mostly for testing vs. setlocale, but I agree it doesn't serve a purpose anymore.

Reply:
Duly noted. I modified the commit message to explain that.